### PR TITLE
Added custom toolbar for iOS

### PR DESCRIPTION
**Pull request by reidmain**

### CODE REVIEW

**Comment by reidmain** [comment]
I originally thought that using the iOS navigation bar was the simplest change to make because all of the other customized UI of this plugin (toolbar, location bar, etc) were hardcoded and using manual frame layouts. I would need to hack in a custom toolbar that still behaved well with everything that existed.

I thought that by using the navigation bar I could just easily show/hide it as needed. Unfortunately that assumption provided poor as all of the manual frame layout seems to be causing issues when you transition from a webpage that needs the custom toolbar to one that does not.

I may need to rethink this solution and move to the painful one of integrating this custom toolbar with all the manual frame layouts that are currently happening.

File: src/ios/CDVWKInAppBrowser.m [hunk 1]
```diff
@@ -257,16 +257,18 @@ - (void)openInInAppBrowser:(NSURL*)url withOptions:(NSString*)options
     
     [self.inAppBrowserViewController navigateTo:url];
     if (!browserOptions.hidden) {
-        [self show:nil withNoAnimate:browserOptions.hidden];
+        [self show:nil withNoAnimate:browserOptions.hidden displayCustomToolbar:browserOptions.customToolbar toolbarTitle:browserOptions.customToolbarTitle toolbarForegroundColor:[CDVWKInAppBrowserViewController colorFromHexString:browserOptions.customToolbarFg] toolbarBackgroundColor:[CDVWKInAppBrowserViewController colorFromHexString:browserOptions.customToolbarBg]];
```
Comment (sundeepgupta) [inline src/ios/CDVWKInAppBrowser.m:260]: Super long line! it's been a while since I've been in ObjC

File: src/ios/CDVWKInAppBrowser.m [hunk 2]
```diff
@@ -1060,7 +1081,7 @@ - (void)viewDidDisappear:(BOOL)animated
 
 - (UIStatusBarStyle)preferredStatusBarStyle
 {
-    return UIStatusBarStyleDefault;
+    return UIStatusBarStyleLightContent;
```
Comment (sundeepgupta) [inline src/ios/CDVWKInAppBrowser.m:1084]: For my own personal learning, why is this needed?

**Review by sundeepgupta — APPROVED** [review]

File: src/ios/CDVWKInAppBrowser.m (hunk 1 above)
Comment (maxkorytko) [inline src/ios/CDVWKInAppBrowser.m:260]: This line is so long. What do you think of breaking it into multiple lines, so that it's easier to read?

File: src/ios/CDVWKInAppBrowser.m [hunk 3]
```diff
@@ -257,16 +257,18 @@ - (void)openInInAppBrowser:(NSURL*)url withOptions:(NSString*)options
...
     }
 }
 
 - (void)show:(CDVInvokedUrlCommand*)command{
-    [self show:command withNoAnimate:NO];
+    [self show:command withNoAnimate:NO displayCustomToolbar:NO toolbarTitle:nil toolbarForegroundColor:nil toolbarBackgroundColor:nil];
 }
 
-- (void)show:(CDVInvokedUrlCommand*)command withNoAnimate:(BOOL)noAnimate
+- (void)show:(CDVInvokedUrlCommand*)command withNoAnimate:(BOOL)noAnimate displayCustomToolbar:(BOOL)displayCustomToolbar toolbarTitle:(NSString *)toolbarTitle toolbarForegroundColor:(UIColor *)toolbarForegroundColor toolbarBackgroundColor:(UIColor *)toolbarBackgroundColor
```
Comment (maxkorytko) [inline src/ios/CDVWKInAppBrowser.m:268]: Same comment as above. This line will grow longer as more parameters are added. I suggest breaking it up into multiple lines.
